examples/s2_field_combination_extern_rf_train/nrw_crop_extern_s2_workflow_to_datasets_with_timeshift.ipynb

```diff
@@ -0,0 +1,2364 @@
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "B02\n",
+      "status=completed, time_started=2023-04-08 18:25:17.424000+00:00, info={'dataset': 'cf14cbe6-d747-48ca-abfe-bd9ea87618a4', 'upload': '26414c0d-e733-4fc8-98fa-7c5a99e91f5d'}, time_total=00:02:20\n",
+      "B03\n",
+      "status=completed, time_started=2023-04-08 18:27:42.743000+00:00, info={'dataset': 'a22225d9-1e96-4646-ad67-0efe41291361', 'upload': '83ee03c6-a7de-44b2-b967-1d2ca5634f76'}, time_total=00:02:22\n",
+      "B04\n",
+      "status=completed, time_started=2023-04-08 18:30:08.149000+00:00, info={'dataset': '47d9ee89-6d87-4228-824d-46c52c751db0', 'upload': '32ebe460-a501-4430-9555-bb0bfe074606'}, time_total=00:02:20\n",
+      "B08\n",
+      "status=completed, time_started=2023-04-08 18:32:33.585000+00:00, info={'dataset': '3db91688-19f2-4dca-9995-0f4097b3996a', 'upload': 'd1403fb3-969e-4171-9d48-053c42947dd3'}, time_total=00:02:19\n",
+      "NDVI\n",
+      "status=completed, time_started=2023-04-08 18:34:53.943000+00:00, info={'dataset': '2a36bca3-fbe9-4d24-854f-fa159b755bfb', 'upload': 'ebb5ec7b-fda1-4bfe-ae19-f2e73fa08c3a'}, time_total=00:03:20\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "{'B02': TaskStatusInfo(status='completed', time_started=datetime.datetime(2023, 4, 8, 18, 25, 17, 424000, tzinfo=datetime.timezone.utc),info = {'dataset': 'cf14cbe6-d747-48ca-abfe-bd9ea87618a4', 'upload': '26414c0d-e733-4fc8-98fa-7c5a99e91f5d'}, time_total = '00:02:20'),\n",
+       " 'B03': TaskStatusInfo(status='completed', time_started=datetime.datetime(2023, 4, 8, 18, 27, 42, 743000, tzinfo=datetime.timezone.utc),info = {'dataset': 'a22225d9-1e96-4646-ad67-0efe41291361', 'upload': '83ee03c6-a7de-44b2-b967-1d2ca5634f76'}, time_total = '00:02:22'),\n",
+       " 'B04': TaskStatusInfo(status='completed', time_started=datetime.datetime(2023, 4, 8, 18, 30, 8, 149000, tzinfo=datetime.timezone.utc),info = {'dataset': '47d9ee89-6d87-4228-824d-46c52c751db0', 'upload': '32ebe460-a501-4430-9555-bb0bfe074606'}, time_total = '00:02:20'),\n",
+       " 'B08': TaskStatusInfo(status='completed', time_started=datetime.datetime(2023, 4, 8, 18, 32, 33, 585000, tzinfo=datetime.timezone.utc),info = {'dataset': '3db91688-19f2-4dca-9995-0f4097b3996a', 'upload': 'd1403fb3-969e-4171-9d48-053c42947dd3'}, time_total = '00:02:19'),\n",
+       " 'NDVI': TaskStatusInfo(status='completed', time_started=datetime.datetime(2023, 4, 8, 18, 34, 53, 943000, tzinfo=datetime.timezone.utc),info = {'dataset': '2a36bca3-fbe9-4d24-854f-fa159b755bfb', 'upload': 'ebb5ec7b-fda1-4bfe-ae19-f2e73fa08c3a'}, time_total = '00:03:20')}"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "status_agg = {}\n",
+    "\n",
+    "for b in [\"B02\", \"B03\", \"B04\", \"B08\", \"NDVI\"]:\n",
```

Comment:
same as above

Reply:
run time is so low i don't think we need to make the notebook more complex using threads.